pathfinder/tests/fixtures/network_service.py

```diff
@@ -1,17 +1,17 @@
 import random
-from typing import List, Callable, Generator
-
+from typing import Callable, Generator, List
 from unittest.mock import Mock, patch
+
```

Comment:
why do you need a new space here?

Reply:
`isort` adds a new line between python packages and third party packages.

Reply:
ah ok thanks got it